src/lib/api/conversions.ts

```diff
@@ -0,0 +1,54 @@
+import { RemoteItem } from '@dcl/builder-client'
```

Comment:
Are we not moving the rest of the (from/to) methods here to reduce the surface of the PR? If so, maybe add a comment ?

Reply:
Cool!